pages/item/_.vue

```diff
@@ -97,69 +97,11 @@
             />
           </div>
 
-          <div>
-            <b-card
-              no-body
-              class="mb-3 rounded-0"
-            >
-              <b-tabs card>
-                <b-tab
-                  :title="$t('record.goodToKnow')"
-                  active
-                >
-                  <b-card-text
-                    text-tag="div"
-                    data-qa="main metadata section"
-                  >
-                    <MetadataField
-                      v-for="(value, name) in coreFields"
-                      :key="name"
-                      :name="name"
-                      :field-data="value"
-                    />
-                  </b-card-text>
-                </b-tab>
-                <b-tab
-                  :title="$t('record.allMetaData')"
-                >
-                  <b-card-text
-                    text-tag="div"
-                  >
-                    <MetadataField
-                      v-for="(value, name) in allMetaData"
-                      :key="name"
-                      :name="name"
-                      :field-data="value"
-                    />
-                  </b-card-text>
-                </b-tab>
-                <b-tab
-                  v-if="Boolean(transcribingAnnotations.length)"
-                  :title="$t('record.transcription')"
-                >
-                  <b-card-text
-                    text-tag="div"
-                  >
-                    <p
-                      class="disclaimer px-2 pb-3 d-flex"
-                    >
-                      {{ $t('record.transcriptionDisclaimer') }}
-                    </p>
-                    <div
-                      v-for="(transcription, index) in transcribingAnnotations"
-                      :key="index"
-                      :lang="transcription.body.language"
-                    >
-                      <p>{{ transcription.body.value }}</p>
-                      <hr
-                        v-if="index !== (transcribingAnnotations.length - 1)"
-                      >
-                    </div>
-                  </b-card-text>
-                </b-tab>
-              </b-tabs>
-            </b-card>
-          </div>
+          <MetadataBox
```

Comment:
The main content & metadata box should take up the full width of the page with a maximum width of 1024 set. The acceptance criteria maybe doesn't explain this well but I believe we need to remove the column containing the relatedEntities and set the main content col to full width on desktop not col-lg-9.

@ashmarriotteuropeana can you confirm this is within scope of this ticket? i.e. removing the relatedEntities column to allow the main content to be full width